SQL1/Homework1-SQL.sql

```diff
@@ -0,0 +1,173 @@
+CREATE DATABASE ZooManagement5;
+
+USE ZooManagement5;
+
+CREATE TABLE Animals1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfAnimal VARCHAR(30),
+    NumberOfAnimal INT,
+);
+
+CREATE TABLE Species1
+(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    NameOfSpecies VARCHAR(20),
+    NumberOfSpecies INT,
+    AnimalsId INT REFERENCES Animals1(Id),

```

Comment:
It's a good thing that you thought of separating the species and the animals in two different tables but it would be easier to work with the database if the Animals table would have a SpeciesId instead of the Species table having am AnimalId.
Also, the foreign key should be named singular, not plural (AnimalId instead of AnimalsId).